torch/onnx/_internal/fx/passes/assertion_removal.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+import torch
+from torch.onnx._internal import _beartype
+from torch.onnx._internal.fx import _pass
+
+
+class RemoveAssertions(_pass.Transform):
+    """This pass removes all assertion and check nodes from the FX graph."""
+
+    _ATEN_ASSERTION_TARGETS = {torch.ops.aten.sym_constrain_range_for_size.default}
```

Comment:
Maybe this one should also be listed in: torch.ops.aten._assert_async.msg?
https://github.com/pytorch/pytorch/issues/112443